uwblueprint/social-diversity-for-children
# Integrate participant info form

### Notion ticket link



[Integrate Participant Information Form With Backend](https://www.notion.so/uwblueprintexecs/086c1b564dbf478faa649a8880e34b81?v=9d3f8002a0d1477ba395616dfc082af7&p=15c0ba7a02cb41ba8e8eb11e25bff1b9)



### Implementation description

- Allows users to enter information in the form fields and saves them to local storage (NOTE: there is currently a bug with rendering the checkboxes/radio buttons correctly on refresh using local storage - potentially a Chakra bug?)
- Upon completion of the form, the data is sent to the backend and saved in the database
- No frontend validation yet - only backend validation



### Steps to test

1. Checkout the branch
2. Login and fill out the form - ensure that you are authenticated!
3. Verify that the request was correctly sent via Postman



### What should reviewers focus on?

-

### Checklist
